introduction/fr/05-principles.asciidoc

```diff
@@ -0,0 +1,70 @@
+Dans les grandes structures, cependant, une telle diffusion peut créer beaucoup de bruit, et il peut être plus approprié de s'assurer que le projet est découvrable dans un outil facile à utiliser.
+Rappelez-vous, l'objectif est de sensibiliser les gens en utilisant les canaux appropriés qui fonctionnent dans VOTRE entreprise.
+
+La liste ci-dessus n'est en aucun cas exhaustive.
+L'ouverture du projet est généralement directement liée à la réussite du projet en termes d'InnerSource.
+Plus il est ouvert, moins il y a de barrières pour les contributeurs potentiels.
+Moins il est ouvert, plus il devient difficile pour quiconque de contribuer.
+
+=== Transparence
+
+Pour que les équipes invitées puissent contribuer de manière significative à un projet, l'équipe hôte doit être _transparente_.
+Cela signifie que les équipes invitées doivent être en mesure de comprendre :
+
+* Le projet/repo et sa direction
+* Les exigences de fonctionnalité en suspens
+* L'avancement des besoins en fonctionnalités
+* La prise de décision de l'équipe hôte
+
+Dans la mesure du possible, les éléments ci-dessus doivent être communiqués clairement et en détail, depuis les définitions internes des équipes jusqu'aux scénarios de cas particuliers propres au projet.
+Cette communication doit être faite d'une manière qui peut être facilement interrogée et comprise par ceux qui ne font pas partie de l'équipe principale.
+
+=== Mentorat hiérarchisé
+
+Le mentorat de l'équipe hôte vers l'équipe invitée via https://innersourcecommons.org/learn/learning-path/trusted-committer/01 [committers de confiance] est un aspect essentiel d'InnerSource.
+https://innersourcecommons.org/learn/learning-path/contributor/01 [Contributeurs] des équipes invitées sont mis à niveau afin qu'ils comprennent suffisamment le projet/repo de l'équipe hôte pour le modifier avec succès.
+Ce faisant, ils en viennent à mieux comprendre le système logiciel de l'équipe hôte en tant que consommateur général et ambassadeur du projet/logiciel.
+Ce contributeur individuel peut, avec le temps et l'expérience, jouer un rôle plus important dans le projet en tant que participant de confiance.
+
+Il est essentiel que l'équipe d'accueil accorde la priorité au mentorat des contributeurs.
+L'équipe hôte doit s'efforcer de prendre le temps d'encadrer les contributeurs invités _au moment où le contributeur en a besoin_ plutôt que lorsque cela lui convient.
+Parfois, il peut s'agir d'un changement de culture pour les ingénieurs de l'équipe hôte qui doivent passer du temps à aider les autres à coder plutôt qu'à coder eux-mêmes.
+Ce mentorat est précieux à la fois pour le contributeur individuel et pour l'hôte, et il vaut la peine de bien le faire.
+Il s'avère mutuellement bénéfique à long terme. En améliorant le code, le contributeur noue ou améliore des relations au sein d'une organisation qui n'aurait peut-être pas pu le faire.
+améliore des relations au sein d'une organisation qui n'auraient peut-être pas existé autrement.
```

Comment:
(bis repetita) to delete "améliore des relations au sein d'une organisation qui n'auraient peut-être pas existé autrement."